### ISSUE TITLE
[CN4J Call] August 10th logistics 

### ISSUE THREAD

**Issue by aeiras** [microprofile/microprofile-marketing#555]
As discussed on todays' call, I volunteer to:

1) create a CN4J minutes space on the MicroProfile Minutes space that allows anyone to contribute to the agenda write-up and keep track of past calls discussions. 

2)  to update the MP calendar event for August the 10th with the new agenda. https://docs.google.com/document/d/1Q3h9dNKSw9xIsCU3ZNPSzzzHhi8Rii50zq7SrTVGA-k/edit#

3) Kevin, would you do the honors to send the agenda with the actual topics up for discussion to the Cn4J forum? 

4) Emily, can you submit a new ticket with the agenda media description so that it can be pushed timely? 

5) Emily can you also make sure that all the documents that are up for discussion, if any, are available under the MP Community CN4J drive (not the EF drive). I found no CN4J documents in the folder that welcome collaboration https://drive.google.com/drive/u/0/folders/1W-vXbSTTB8rwY0gKwgeg_Q8jlv6YlQip

Thank you,

**Comment by aeiras** [comment]
@kwsutter & @Emily-Jiang

**Comment by kwsutter** [comment]
@aeiras, MP does not own the content for these CN4J calls.  This has been a joint effort between @jclingan and Will Lyons.  On today's hangout, I just mentioned that there is a CN4J call scheduled for next week and put it in the minutes.  I plan to participate on these CN4J calls, but I am not driving them.  Thank you.

**Comment by Emily-Jiang** [comment]
+1 on what @kwsutter said. I think John might put an agenda together with Will when the time is closer. At today's call, I just commented that I would like to suggest a topic at CN4J. As you might remember, I suggested once before but it was postponed. Therefore, I don't think it makes sense for me to define the agenda and MP tweets about it.

**Comment by aeiras** [comment]
I am in 100% agreement.

We need the very base, the agenda document in the calendar to start somewhere now that the Messaging has been completed and we can move to other regular topics. 

The space for the agenda collaboration is there. We have the CN4J forum where the agenda ought to be shaped.  
 I added 2 layers to the agenda topics.  I believe the 2 communities need to decide how to coordinate moving forward as to where to place "WIP" documents, etc. 

What I wish never to see again is a CN4J call being set under the private Steering cmtee call and just inform the MP or Jkta community. We must do better and respect all our members. With that hope, I have pushed for some formally that needs discussion next week.
